[Merge-Queue] Skip RemoveFlagsOnPatch if no patch defined

JonWBedard opened the pull request:
#### 55fe4a944835e9fc03e68acade411e7662be8fc5
<pre>
[Merge-Queue] Skip RemoveFlagsOnPatch if no patch defined
<a href="https://bugs.webkit.org/show_bug.cgi?id=238613">https://bugs.webkit.org/show_bug.cgi?id=238613</a>
&lt;rdar://problem/91111541 &gt;

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(RemoveFlagsOnPatch.getResultSummary): Use default summary for skip case.
(RemoveFlagsOnPatch.doStepIf): Do step if patch_id is available.
(RemoveFlagsOnPatch.hideStepIf): Hide step if skipped.


Canonical link: <a href="https://commits.webkit.org/249069@main">https://commits.webkit.org/249069@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@292162">https://svn.webkit.org/repository/webkit/trunk@292162</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

aj062 commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -1710,10 +1709,18 @@ def start(self):
         return None
 
     def getResultSummary(self):
+        if self.results == SKIPPED:
```
Better to not have this handling of SKIPPED case as I explained in https://github.com/WebKit/WebKit/pull/231#discussion_r839863895

JonWBedard committed:
[Merge-Queue] Skip RemoveFlagsOnPatch if no patch defined
https://bugs.webkit.org/show_bug.cgi?id=238613
<rdar://problem/91111541>

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(RemoveFlagsOnPatch.getResultSummary): Use default summary for skip case.
(RemoveFlagsOnPatch.doStepIf): Do step if patch_id is available.
(RemoveFlagsOnPatch.hideStepIf): Hide step if skipped.
